quora-service/src/main/java/com/upgrad/quora/service/dao/UserDao.java

```diff
@@ -21,7 +21,7 @@ public UserEntity createUser(UserEntity userEntity) {
 
     public UserEntity getUser(final String userUuid) {
         try {
-            return entityManager.createNamedQuery("userByUuid", UserEntity.class).setParameter("userUuid", userUuid).getSingleResult();
+            return entityManager.createNamedQuery("userByUuid", UserEntity.class).setParameter("uuid", userUuid).getSingleResult();
```

Comment:
sorry i mis read it but i think it is correct to have userUuid as it is being used this way in JWT

Reply:
Please look at the parameter defined in the NamedQuery "userByUuid"  in UserEntity
@NamedQuery(name = "userByUuid", query = "select u from UserEntity u where u.uuid =:uuid")

Reply:
The above change is in accordance with what has been specified in UserEntity